Formula/p/python@3.12.rb

```diff
@@ -466,23 +469,20 @@ def sitecustomize
   def caveats
     <<~EOS
       Python has been installed as
-        #{HOMEBREW_PREFIX}/bin/python#{version.major_minor}
+        #{HOMEBREW_PREFIX}/bin/python3
 
-      Unversioned and major-versioned symlinks `python`, `python3`, `python-config`, `python3-config`, `pip`, `pip3`, etc. pointing to
-      `python#{version.major_minor}`, `python#{version.major_minor}-config`, `pip#{version.major_minor}` etc., respectively, have been installed into
+      Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
+      `python3`, `python3-config`, `pip3` etc., respectively, have been installed into
         #{opt_libexec}/bin
 
       You can install Python packages with
-        pip#{version.major_minor} install <package>
+        pip3 install <package>
       They will install into the site-package directory
         #{HOMEBREW_PREFIX}/lib/python#{version.major_minor}/site-packages
```

Comment:
Does this work anymore with the externally managed thing?

Reply:
Doc update is being worked on here: https://github.com/Homebrew/brew/pull/16643

Reply:
Doc update is done. I went for a caveats cleanup: if people use pip incorrectly there will be a message about virtualenvs and the externally managed process IMHO.